payload/reggae/rules/d.d

```diff
@@ -115,21 +117,24 @@ Target[] dlangObjectFilesPerPackage(in string[] srcFiles,
     }
     return srcFiles.byPackage.map!(a => Target(a[0].packagePath.objFileName,
                                                command(a),
-                                               a.map!(a => Target(a)).array)).array;
+                                               a.map!(a => Target(a)).array,
+                                               implicits)).array;
 }
 
 /// Generate object files for D sources, compiling each module separately
 Target[] dlangObjectFilesPerModule(in string[] srcFiles,
                                    in string flags = "",
                                    in string[] importPaths = [],
                                    in string[] stringImportPaths = [],
```

Comment:
`in`

Reply:
As in `in Target[]`? That doesn't seem to work as the rest of reggae uses non-cost `Target`.